tools/log-collector/log_collector.go

```diff
@@ -358,31 +377,37 @@ func (l *LogCollector) zipDir() error {
 func (l *LogCollector) filterResourceObjects(resourcePath string,
 	resource *apiv1.APIResource) (allObjects unstructured.UnstructuredList, err error) {
 
+	if l.checkIfMatchesInputGVKs(resource) {
+		log.Infof("Fetching '%s' Resource", resource.Kind)
+		return l.getResourceObjects(resourcePath, resource), nil
+	}
+
 	if (!resource.Namespaced && clusteredResources.Has(resource.Kind)) ||
 		(resource.Namespaced && !excludeResources.Has(resource.Kind)) {
 
-		log.Infof("Fetching '%s' Resource", resource.Kind)
-		allObjects = l.getResourceObjects(resourcePath, resource)
-
 		if resource.Name == CRD {
-			allObjects, err = filterTvkSnapshotAndCSICRD(allObjects)
+			log.Infof("Fetching '%s' Resource", resource.Kind)
```

Comment:
change level to `Debug` .. do the same for other low level information logs.

Reply:
done